src/resmom/linux/mom_mach.c

```diff
@@ -4568,17 +4568,18 @@ static const char *physmem(
     }
 
   mem_total = 0;
-
+  int test_file = 0;
```

Comment:
Variable declarations should be all together at the top of the function, not interspersed throughout.

But...